quickwit/quickwit-integration-tests/src/tests/ingest_v2_tests.rs

```diff
@@ -359,14 +359,17 @@ async fn test_ingest_v2_high_throughput() {
     let line = json!({"body": "my dummy repeated payload"}).to_string();
     let num_docs = body_size / line.len();
     let body = std::iter::repeat_n(&line, num_docs).join("\n");
-    let ingest_resp = ingest(
-        &sandbox.rest_client(QuickwitService::Indexer),
-        index_id,
-        IngestSource::Str(body),
-        CommitType::Auto,
-    )
-    .await
-    .unwrap();
+    let ingest_resp = sandbox
+        .rest_client(QuickwitService::Indexer)
+        .ingest(
+            index_id,
+            IngestSource::Str(body),
+            Some(5_000_000),
```

Comment:
Casually observing. 

Our batches tend to have 200 - 1000 items during bulk request and target 1 - 10 different indexes. The higher end is about 3mb. Here the typical rate of ingest is about 900mb/s 

Our retry work load is capped to 500 items per batch and sits around 1mb. These would appear to touch 1 index. We are trying to determine if they are all the same index

Reply:
I'm playing with the batch size in this test because I had a surprising 500 (internal timeout) error. We want the system to behave consistently regardless of the batch size (obviously with very small batches adding a bit of overhead).

Reply:
What do we consider very small?

Reply:
One overhead of the top of my head is the http frame overhead. If you have more http headers than document payload, that's a bit of unnecessary extra work.

Reply:
the 1MB granularity should be pretty good (negligeable framing overhead in network exchanges but small enough to be nicely balanced between shards)